### ISSUE TITLE
Definir Formato

### ISSUE THREAD

**Issue by Lucasktrindade** [riocss/riocss#1]
Aqui vamos discutir sobre como definir Formato do Meetup do CSS

**Comment by lucianobarauna** [comment]
- Termos um código de conduta
- Sempre se apresentar antes das palestras

**Comment by Lucasktrindade** [comment]
Isso ai deve ser independente de como seja o formato. Eu curti o que o Afonso falou que o pessoal faz em Floripa. Uma palestra, e um intervalo de 1h pra galera conversar sobre. Acho que podemos até diminuir o tempo para uns 30~40 min.

**Comment by Lucasktrindade** [comment]
Outro formato legal é ter uma palestra, e depois os assuntos ficarem em torno disso. A galera podia colar os post-it em algum lugar e depois alguém assumia o assunto.

**Comment by lucianobarauna** [comment]
Acho bem legal e diferente isso, se for assim mesmo e conseguirmos manter eu topo.
Na medida que for avançando em mais eventos vamos atualizando aqui.
@Lucasktrindade @afonsopacifer  e @MathMesquita 
Começamos assim então ?

**Comment by MathMesquita** [comment]
Achei a idéia sobre a palestra + intervalo uma boa, finalizando com essa parada dos post-its para próximos tópicos e tal... pelo o que vejo da Rio.JS o problema geralmente é a definição do tópico, que só é passado na ultima hora.

**Comment by lucianobarauna** [comment]
é poderíamos colocar isso no decorrer do mês até o evento seguinte.
Os galera poderia se candidatar e levar o tema para lá. Se fosse o caso de ter muito tema abriríamos  para votação.

**Comment by Lucasktrindade** [comment]
Sim @MathMesquita curti essa ideia de Post-it para os próximos eventos. Podemos abrir uma issue aqui pro próximo evento e a galera ir discutindo

**Comment by MathMesquita** [comment]
Ja é uma boa iniciativa, nós vamos criar um repo diferente pra cada evento ?

**Comment by lucianobarauna** [comment]
Acho que não seria o caso.
Poderíamos colocar dentro de um único repo chamado eventos e ali teríamos uma estrutura de pasta e dentro dela um readme.md com os links e informações o que rolou no evento e lá para frente poderíamos ter isso num blog maroto o que acham ?

**Comment by Lucasktrindade** [comment]
Sim, pode ser. Ou melhor ter um medium com as comunidades de Frontend do Rio. Igual a galera faz com o Android Dev BR no medium https://medium.com/android-dev-br

**Comment by lucianobarauna** [comment]
Gostei da ideia, mais seria um medium geral ou só do Rio.css ?

**Comment by Lucasktrindade** [comment]
Acho que pode ser um geral, e vamos chamando a galera pra participar e dando acesso aos outros meetups de Front que vem a surgir no Rio

**Comment by lucianobarauna** [comment]
Pode ser de medium eu entendo pouco na real.
Ainda vou começar a mexer com ele daqui uns dias.

**Comment by MathMesquita** [comment]
Vou criar issue de fazer o medium então @Lucasktrindade @lucianobarauna

**Comment by lucianobarauna** [comment]
Fechamos o formato ?
Se fechamos podemos criar um formatoEncontro.md, descrever tudo nele e depois colocar no repositório e quando tivermos alguma melhoria discutimos e atualizamos ele. O que acham ?

**Comment by afonsopacifer** [comment]
Fala pessoal rsrsr vim contribuir (antes tarde do que nunca) rs.

Como o @Lucasktrindade deve ter comentado, conversei com o Maujor p ele falar no primeiro evento, e eles mesmo sugeriu esse lance de dar a palestra de forma mais informal e ir trocando ideia durante a palestra e discutirmos sobre depois :)

Tem um detalhe que temos que nos atentar, principalmente agora no "reta final", temos que ver como fica a politica de "Confirmar presença e não ir".. Minha sugestão e deixar claro no texto do meetup alguma regra sobre (como suspender a inscrição p um próximo meetup como "pena", como fazem em SP). O que acham?

Tbm precisamos mandar e-mail para o pessoal que se inscreveu no Meetup pedindo p liberarem a vaga caso não possam ir.. pq se não pedirmos p fazerem isso, o pessoal simplesmente se inscreve por se inscrever (e geralmente nunca pensou em ir, só se inscreve de fogo mesmo).

Eu posso mandar esse e-mail p passoal via Meetup (só preciso de acesso) ou ajudar quem for enviar (pq já fiz isso nos eventos que ajudei).

**Comment by afonsopacifer** [comment]
Ahhh , precisamos tbm limitar as inscrições no meetup... p elas ficarem como encerradas.. isso gera credibilidade e mostra que não estamos brincando com relação a "quem falou que vai, ou vai, ou avisa que vai rs"

PS: Tudo o que to falando são sugestões, baseado nos moldes de sucesso de SP <3

Edit: Podemos usar a lista de espera do Meetup p gerenciar as vagas e quem realmente vai :)

**Comment by Lucasktrindade** [comment]
Acho que consigo te colocar lá no meetup como organizador e quanto a quantidade de vagas, podemos limitar sim até pra fazer algo como planejamos de querer acolher todos e sim acho que gera credibilidade.

**Comment by MathMesquita** [comment]
voto pela limitação 👍

**Comment by lucianobarauna** [comment]
Ok limitar é uma boa mesmo

**Comment by lucianobarauna** [comment]
Criei um pré readme do formato para as coisas que ficarem definidas aqui vamos colocando lá para pubicarmos depois.
A issue é a #17 
Agora fica em questão de regra para quem dizer que vai e não for.
Como vamos fazer esse controle independente do lugar que for realizado o evento ?
